python-backend/app/api/permits/permit/models/permit.py

```diff
@@ -17,9 +17,7 @@ class Permit(AuditMixin, Base):
     permit_status_code = db.Column(
         db.String(2), db.ForeignKey('permit_status_code.permit_status_code'))
     permit_status_code_relationship = db.relationship(
-        'PermitStatusCode',
-        foreign_keys=[permit_status_code],
-        lazy='select')
+        'PermitStatusCode', foreign_keys=[permit_status_code], lazy='select')
```

Comment:
didn't know about this foreign_keys attribute, do we want to use this going forward?

Reply:
We may want to, but it doesn't actually do anything in this use. I actually just removed it.